hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/IncrementalRelation.scala

```diff
@@ -225,6 +236,9 @@ class IncrementalRelation(val sqlContext: SQLContext,
           }
         }
 
+        if (!requiredColumns.contains(HoodieRecord.COMMIT_TIME_METADATA_FIELD)) {
```

Comment:
Add a code comment here that "remove the COMMIT_TIME_METADATA_FIELD if not requested"

Reply:
Ack.